alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/MonitorRunResult.kt

```diff
@@ -77,6 +77,10 @@ data class TriggerRunResult(
     val actionResults: MutableMap<String, ActionRunResult> = mutableMapOf()
 ) : ToXContent {
     override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
+        var msg = error?.message;
```

Comment:
I believe we never use this in the ```XContentBuilder``` after assignment

Reply:
Right.
missed online during copy paste from my working repo to repo for PR.
my bad.